xbmc/settings/windows/GUIControlSettings.cpp

```diff
@@ -591,43 +609,46 @@ void CGUIControlListSetting::Update(bool updateDisplayOnly /* = false */)
 
   m_pButton->SetLabel2(label2);
 
-  // disable the control if
-  // * there are no items to be chosen
-  // * only one value can be chosen and there are less than two items available
-  if (!m_pButton->IsDisabled() &&
-     (options.Size() <= 0 ||
-     (!control->CanMultiSelect() && options.Size() <= 1)))
-    m_pButton->SetEnabled(false);
+  if (!updateDisplayOnly)
+  {
+    // disable the control if
+    // * there are no items to be chosen
+    // * only one value can be chosen and there are less than two items available
+    if (!m_pButton->IsDisabled() &&
+      (options.Size() <= 0 ||
+      (!control->CanMultiSelect() && options.Size() <= 1)))
+      m_pButton->SetEnabled(false);
+  }
 }
 
-bool CGUIControlListSetting::GetItems(SettingConstPtr setting, CFileItemList &items) const
+bool CGUIControlListSetting::GetItems(SettingConstPtr setting, CFileItemList &items, bool updateItems) const
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting


> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
Fixed